influunt-app/app/views/controladores/index.html

```diff
@@ -1,77 +1,83 @@
-  </div>
-  <div class="col-lg-10 col-sm-10">
-  <section role="complementary" class="">
-      <div class="col-lg-4 col-sm-4 no-padding">
-        <form role="form" class="form-group">
-          <div class="input-group">
-            <input type="text" class="form-control" ng-model="filtros.campo_busca">
-            <span class="input-group-btn">
-              <button type="button" data-toggle="tooltip" data-placement="top" title="Pesquisar" class="btn btn-primary">
-                <i class="fa fa-search" aria-hidden="true"></i>
-              </button>
-            </span>
           </div>
-        </form>
-      </div>
-      <!--
-      <div class="col-lg-2 col-sm-2">
-        <button type="button" class="btn btn-primary"><i class="fa fa-th-large" aria-hidden="true"></i></button>
-        <button type="button" class="btn btn-primary"><i class="fa fa-th-list" aria-hidden="true"></i></button>
-      </div>
-      -->
-      <div class="clearfix"></div>
-      <div class="col-lg-2 no-padding">
-        <button type="button" class="btn btn-primary container-fluid" data-toggle="tooltip" data-placement="top" title="Adicionar novo controlador" ui-sref="app.wizard_controladores.dados_basicos">Novo controlador</button>
-      </div>
-      <div class="clearfix"></div>
-      <div class="col-lg-12 controller no-padding">
-        <ul>
-          <li ng-repeat="controlador in lista | filter: filtros.campo_busca | checkboxFilter:filtroLateral">
-            <div class="ibox float-e-margins">
-              <div class="ibox-title">
-                <h5>{{ controlador.CLC }}</h5>
-              </div>
-              <div class="ibox-content">
-                <p class="title">{{ controlador.localizacao }}</p>
-                <!--
-                <p class="status"><b>Status:</b></p>
-                <p class="status-active"><i class="fa fa-circle text-navy"></i>Ativo</p>
-                -->
-                <hgroup>
-                  <a class="btn btn-primary" data-toggle="tooltip" data-placement="top" title="Editar controlador" type="button" ui-sref="app.wizard_controladores.dados_basicos({id: controlador.id})">
-                    <i class="fa fa-pencil"></i>
-                  </a>
-                  <a class="btn btn-primary" data-toggle="tooltip" data-placement="top" title="Vizualizar controlador" type="button" ui-sref="app.controladores_show({id: controlador.id})">
-                    <i class="fa fa-eye"></i>
-                  </a>
-                  <a class="btn btn-primary" data-toggle="tooltip" data-placement="top" title="Vizualizar planos" type="button" ui-sref="app.planos({id: controlador.id})">
-                    <i class="fa fa-file-text"></i>
-                  </a>
-                </hgroup>
+          <div class="ibox-content">
+            <div class="row">
+              <div class="col-sm-3 m-b-xs"><select class="input-sm form-control input-s-sm inline">
+                <option value="">Escolha a Área</option>
```

Comment:
Todo o cabeçalho está fake. Vamos criar uma issue para o redesign completo
dessa tela.

On Thursday, July 28, 2016, leonardoherbert <notifications@github.com>
wrote:

> In influunt-app/app/views/controladores/index.html
> <https://github.com/influunt/influunt/pull/373#discussion_r72708160>:
>
> > -                -->
> > -                <hgroup>
> > -                  <a class="btn btn-primary" data-toggle="tooltip" data-placement="top" title="Editar controlador" type="button" ui-sref="app.wizard_controladores.dados_basicos({id: controlador.id})">
> > -                    <i class="fa fa-pencil"></i>
> > -                  </a>
> > -                  <a class="btn btn-primary" data-toggle="tooltip" data-placement="top" title="Vizualizar controlador" type="button" ui-sref="app.controladores_show({id: controlador.id})">
> > -                    <i class="fa fa-eye"></i>
> > -                  </a>
> > -                  <a class="btn btn-primary" data-toggle="tooltip" data-placement="top" title="Vizualizar planos" type="button" ui-sref="app.planos({id: controlador.id})">
> > -                    <i class="fa fa-file-text"></i>
> > -                  </a>
> > -                </hgroup>
> > +          <div class="ibox-content">
> > +            <div class="row">
> > +              <div class="col-sm-3 m-b-xs"><select class="input-sm form-control input-s-sm inline">
> > +                <option value="">Escolha a Área</option>
>
> Por enquanto as áreas vão ser fixas? Se for vamos criar uma issue para
> resolver isso depois.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/influunt/influunt/pull/373/files/26375e3543edb5bd9f5de46b30e50161d02bd747#r72708160>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAGDrZvv2HfaSwg3_TR8c4x9IV9Hz6wSks5qaSGtgaJpZM4JXqnA>
> .
>


-- 
Rodrigo Sol

rodrigo@rarolabs.com.br
http://www.rarolabs.com.br